ProcessOut/Classes/TokenRequest.swift

```diff
@@ -8,15 +8,21 @@
 import Foundation
 
 class TokenRequest: Encodable {
+    var customerID: String = ""
+    var tokenID: String = ""
     var source: String = ""
     var verify = true
     var enable3DS2 = true
     var thirdPartySDKVersion: String = ""
+    var preferredScheme: String = ""
     enum CodingKeys: String, CodingKey {
+        case customerID = "customer_id"
+        case tokenID = "token_id"
         case source = "source"
         case verify = "verify"
         case enable3DS2 = "enable_three_d_s_2"
         case thirdPartySDKVersion = "third_party_sdk_version"
+        case preferredScheme = "preferred_scheme"
     }
     
     init(source: String, thirdPartySDKVersion: String) {
```

Comment:
should I add all new fields also here? @simeon-kostadinov-cko

Reply:
This is sort of a constructor (initialiser) so I guess we can add them and it will help set the class fields.